src/main/java/com/exactpro/th2/fix/client/Main.java

```diff
@@ -224,8 +224,7 @@ public static void run(Settings settings, MessageRouter<MessageGroupBatch> messa
         });
 
         Event rootEvent = MessageRouterUtils.storeEvent(eventRouter, Event.start()
-                        .description("Root event")
-                        .name(boxName + " " + Instant.now())
+                        .name("Sessions events " + Instant.now())
                         .type("Microservice")
                         .bodyData(FixBeanUtil.getSessionTable(settings.getSessionSettings()))
                 , null);
```

Comment:
> null

Didn't we decide to use root-event from common as a parent for this event?

Reply:
we discussed this issue with @alexander-zhilov and decided not to do additional nesting for comfortability purposes

Reply:
So we're okay with 2 root-events? =/

Reply:
We do not have options to remove or edit a root event generated by CommonFactory.
Also i think that one more event nesting is inconvenient.
Let it be a two root events as temporary solution, we add a new feature to modify CommonFactory root event in the future.